Marlin/Marlin_main.cpp

```diff
@@ -3581,7 +3582,6 @@ inline void gcode_G4() {
   inline void gcode_G10() {
     #if EXTRUDERS > 1
       const bool rs = parser.boolval('S');
-      retracted_swap[active_extruder] = rs; // Use 'S' for swap, default to false
```

Comment:
Deleted because swap status is needed to know the current state 
If G10 force this , impossible to know what was the old state
Now , G10 give instructions and FWretract function make the job correctly